fastestimator/estimator.py

```diff
@@ -97,6 +99,7 @@ def __init__(self,
                              total_epochs=epochs,
                              train_steps_per_epoch=train_steps_per_epoch,
                              eval_steps_per_epoch=eval_steps_per_epoch,
+                             eval_log_steps=eval_log_steps,
```

Comment:
where does this variable get used within the estimator?

Reply:
in particular, why are we allowing the user to pass this value here when line 349 unilaterally overwrites the value stored in the system object?